docs/source/conf.py

```diff
@@ -60,9 +60,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '3.3.3-beta'
+version = '4.0.0'
 # The full version, including alpha/beta/rc tags.
-release = '3.3.3-beta'
+release = '4.0.0-1'
```

Comment:
The -N suffixes are for package releases, not the source code.

Reply:
Well yes but conf.py has been like this for as long as I know. Also the intention is to merge this as beta then update to release.... I can fix this now if you think it would be better :-D